docs/vocascan-server/pm2/pm2.md

```diff
@@ -6,36 +6,64 @@
    sudo apt install nodejs npm git
    ```
 
-2. Install pm2
+   Check if nodejs version is at least v14.x.x
```

Comment:
sorry, that I said it wrong, but I think 12.x.x should also work as it is described in our [package.json](https://github.com/vocascan/vocascan-server/blob/main/package.json#L47)